app/requirements.txt

```diff
@@ -1,6 +1,7 @@
 -e git+https://github.com/linsomniac/python-memcached.git@37f55ca4ad94ca4ade30d6be28e1facb79ac3182#egg=python-memcached
```

Comment:
Yeah, we need this commit:

https://github.com/linsomniac/python-memcached/commit/94089e66cd0acca36d5206c7e62499bf12a502d6